tests/nosetests/pyanaconda_tests/module_localization_test.py

```diff
@@ -128,21 +128,6 @@ def set_language_kickstarted_test(self):
         self.assertEqual(self.localization_interface.LanguageKickstarted, True)
         self.callback.assert_called_once_with(LOCALIZATION.interface_name, {'LanguageKickstarted': True}, [])
 
-    @patch('pyanaconda.dbus.DBus.publish_object')
-    def install_language_with_task_test(self, publisher):
-        """Test InstallLanguageWithTask."""
-        self.localization_interface.SetLanguage("cs_CZ.UTF-8")
-        task_path = self.localization_interface.InstallLanguageWithTask("/")
```

Comment:
Please, just modify the test for `InstallWithTasks`.

Reply:
Oh, good point, that's really simple - added. :)